src/extension/settingsHelper.ts

```diff
@@ -63,4 +67,16 @@ export class SettingsHelper {
                 return settingsJson["typescript.tsdk"] || "";
             });
     }
+
+    /**
+     * Reads and parses the launch.json file.
+     */
+    public static readLaunchJson(): Q.Promise<any> {
```

Comment:
We read JSON files in a few different places. I'd refactor this to take the file path as a parameter, and put a generic readJson method in the FileSystem class.